packages/hippodrome-ui/src/components/transaction/AwaitingMint.tsx

```diff
@@ -233,17 +234,16 @@ const DepositStakeConfirmed: React.FC<DepositConfirmedProps> = ({
     try {
       setLoading(true)
       console.log("swapping: ", deposit)
-      await doAddLiquidity(deposit, deposit.lockAndMint.params)
+      await doAddLiquidity(chain, deposit, deposit.lockAndMint.params)
       history.push("/")
       toast({
-        title: "Stake completed",
-        description: "Your stake will appear in next screen",
-        duration: 9000,
```

Comment:
this will default to 5000 if it isn't specified

Reply:
ah ok - well i'll add back in a *longer* duration then in another PR